SWARAM-UI-Documentation/index.html

```diff
@@ -0,0 +1,460 @@
+                            <button class="btn btn-primary-text font-size-6 text-gray" >ADD TO WISHLIST</button>
+                        </div>
+                 </div>
+                 <div class="card-content-padding font-size-4 ">
+                     Rs 899 
+                 </div>
+            </div>
+        </div>
+        <div class="code-div">
+            <embed src="https://carbon.now.sh/embed/GlPYBRS7dzxnWTbthX9r" width="100%"></embed>
+        </div>     
+    </section>
+
+    
+    <section id="list">
+        <h1 class="heading-large heading">List</h1>
+
+        <h3 class="heading">Spaced List Items</h3>
+        <div class="center-div">
+            <ul class="spaced-list-item-group" style="width:14rem">
+                <li class="spaced-list-item">Favourites</li>
+                <li class="spaced-list-item">Playlist</li>
+                <li class="spaced-list-item">Subscriptions</li>
+                <li class="spaced-list-item">Trending</li>
+                <li class="spaced-list-item">Watch Later</li>
+            </ul>
+        </div>
+       
+        <div class="code-div">
+            <embed src="https://carbon.now.sh/embed/tFZgMu4uTchUPUL2A6BJ" width="100%"></embed>
+        </div> 
+    
+        <h3 class="heading">Stacked List Items</h3>
+        <div class="center-div">
+        <div class="stacked-list-group" style="width:23rem ;max-width: 90%;">
+            <div class="stacked-item">
+                <div class="img-margin">
+                    <img class="round-img img-size-small" src="https://tinyurl.com/2fzxkc44">
+                </div>
+                <div>
+                   <span class="font-bold-1 font-size-5">Janaki</span> started following you
+                    <span class="text-gray">1d</span>
+                </div>
+            </div>
+            <div class="stacked-item" style="display: flex;">
+                <div class="img-margin">
+                    <img class="round-img img-size-small" src="https://tinyurl.com/2fzxkc44">
+                </div>
+                <div>
+                   <span class="font-bold-1 font-size-5">Janaki</span> liked your message
+                    <span class="text-gray">1d</span>
+                </div>
+            </div>            
+        </div>
+        </div>
+
+        <div class="code-div">
+            <embed src="https://carbon.now.sh/embed/PBIPc6S08mPCb52mnFzO" width="100%"></embed>
+        </div>  
+    </section>
+   
+  
+    <section id="input">
+    <h1 class="heading-large heading">Input</h1>
+    <div class="center-div">
+        <div style="width:20rem">
+            <div class="flex-column">
+                <label class="font-size-6 font-bold-1">Name</label>
+                <input type="text" class="text-input">
+            </div>
+            <div class="flex-column">
+                <label class="font-size-6 font-bold-1">Email</label>
+                <input type="text" class="text-input">
```

Comment:
You can use <input type="email"> so that the input can be automatically validated to be a properly formatted email address.
You can refer this for more info : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email

As a Side note  you can also add a submit button to your form.

Reply:
Great Suggestion. Have changed it.Thank you.